### ISSUE TITLE
make 'vak prep' command work with PREDICT section in config

### ISSUE THREAD

**Issue by NickleDave** [NickleDave/vak#82]
Currently if durations for train / val / test sets are not specified, then `prep` assumes the entire data directory is a training set -- which is a problem when it's the data for which you want to predict annotation.

New behavior should be to identify whether `config.ini` has TRAIN, LEARNCURVE or PREDICT section -- if there are no train / val / test set durations, and there's a TRAIN section, assume it's TRAINing data, and if there's a PREDICT section, assume it's PREDICTion data

**Comment by NickleDave** [comment]
done in #83
